dataland-backend/backendOpenApi.json

```diff
@@ -2847,6 +2847,16 @@
         }
       },
       "LksgGeneral": {
+        "required": [
+          "dataDate",
+          "listOfProductionSites",
+          "lksgInScope",
```

Comment:
Shoulnd't there be some kind of scroll if I click the button in case of an error?

Reply:
After update of FormKit, the submit Button now seems to correctly scroll to the missing fields.

Reply:
It seems to not work on a re-click

Reply:
Yes, apperently Scroll to invalid was never set up for the company upload form. This has now been changed and it seems to work.

Reply:
ok